RtEthSample/RtEthSample.inx

```diff
@@ -271,8 +287,11 @@ IMLow                    = "Low"
 IMMedium                 = "Medium"
 TransmitBuffers          = "Transmit Buffers"
 
-RTL8168.DeviceDesc       = "Realtek PCIe GBE Family Controller NetAdapter Sample Driver"

```

Comment:
Realized that the DeviceDesc and DisplayName should not have been changed. Fixed it.